crates/sui-graphql-rpc/src/types/move_type.rs

```diff
@@ -29,7 +29,7 @@ type MoveTypeSignature =
   | \"u8\" | \"u16\" | ... | \"u256\"
   | { vector: MoveTypeSignature }
   | {
-      struct: {
+      datatype: {
```

Comment:
Thoughts on `datatype` vs `dataType`? (I don't mind really, but the thought came to me because I would naturally have called the Rust type `DataType` and that would have been converted to `dataType` by serde, so I was curious if there was a compelling reason to go against the grain on that).

Reply:
I went with `datatype` and `Datatype` as the words in Rust actually! I played with `DataType` as well, but wound up settling for this since otherwise it leads to (IMO) annoying naming everywhere, e.g., 

```
DataTypeTypeParameters
data_type_handles
data_type_handle_at
```
(etc)

whereas treating it as one word makes these a bit nicer and makes things (IMO)

```
DatatypeTypeParameters
datatype_handles
datatype_handle_at 
```
(etc)